src/harness/certs/README.md

```diff
@@ -12,19 +12,24 @@ WINNF-15-S-0065" document. Naming and base configuration are issued from
 https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/tree/master/cert
 
 ```
-                               root_ca ------------------------------------------------------                
-                               /     \          					     \                 
-                              /       \          					      \            
-                         sas_ca       cbsd_ca   					   proxy_ca
-                         /    |          \              	  			 	\
-                        /     |           \              	  				 \
-             admin_client  server  client|device_[a|c]|corrupted_client|wrong_type_client|   	domain_proxy
-				   	               client_expired|client_inapplicable 
-   unrecognized_ca              non_cbrs_root_ca
-         |                           |
-  unrecognized_device        non_cbrs_root_signed_cbsd_ca
-                                     |
-                             non_cbrs_signed_device
+                   root_ca --------------------------------------
+                   /     \          			         \                 
+                  /       \          			          \            
+             sas_ca       cbsd_ca   			       proxy_ca
+             /    |          \              	  	            \
+            /     |           \              	  	             \
+ admin_client  server  client|device_[a|c]|corrupted_client      domain_proxy
+			wrong_type_client|client_expired         corrupted_domain_proxy
+                        client_inapplicable                      wrong_type_domain_proxy
+                                                                 domain_proxy_expired
+                                                                 domain_proxy_inapplicable
+
+
+   unrecognized_ca       non_cbrs_root_ca----------------------
```

Comment:
`unrecognized_ca` is currently named `unrecognized_root_ca`.
Also add the `unrecognized_domain_proxy` under `unrecognized_device`, here in the schema and below in the description

Reply:
thanks for the findings, changed accordingly. Please take a look again.